config/default/common/config/metadata/layers/multi-mission/hls/HLS_L30_Nadir_BRDF_Adjusted_Reflectance.md

```diff
@@ -0,0 +1,9 @@
+The Reflectance (Nadir BRDF Adjusted, v1.5) imagery layer from Landsat 8/OLI i is a true-color or natural color image because this combination of wavelengths is similar to what the human eye would see. It consists of natural-looking images of land surface, oceanic and atmospheric features. The Reflectance (Nadir BRDF Adjusted, v1.5) imagery layer from Landsat 8/OLI provides 30m Nadir Bidirectional Reflectance Distribution Function (BRDF)-Adjusted Reflectance (NBAR) and is derived from the joint NASA/USGS Landsat 8 Operational Land Imager (OLI) data product through the Harmonized Landsat and Sentinel-2 (HLS) project.
```

Comment:
Possible extra "i":
```suggestion
The Reflectance (Nadir BRDF Adjusted, v1.5) imagery layer from Landsat 8/OLI is a true-color or natural color image because this combination of wavelengths is similar to what the human eye would see. It consists of natural-looking images of land surface, oceanic and atmospheric features. The Reflectance (Nadir BRDF Adjusted, v1.5) imagery layer from Landsat 8/OLI provides 30m Nadir Bidirectional Reflectance Distribution Function (BRDF)-Adjusted Reflectance (NBAR) and is derived from the joint NASA/USGS Landsat 8 Operational Land Imager (OLI) data product through the Harmonized Landsat and Sentinel-2 (HLS) project.
```

Reply:
Good eye. Fixed!